### DISCUSSION TITLE
Problem with theme!

### DISCUSSION THREAD

**Discussion by ErfanRez** [saadeghi/daisyui#2588] [Q&A] [answered]
I have set up the theme as explained in the docs, but the problem is on smaller screens there is still a dark layer under the main theme.
![Screenshot (179)](https://github.com/saadeghi/daisyui/assets/100679623/787443fe-4ad8-45b5-b9fc-277892540ada)
![Screenshot (181)](https://github.com/saadeghi/daisyui/assets/100679623/43a9aab2-9b08-495d-a8c3-d94bf7c624f7)

**Comment by saadeghi** [comment]
Do you have other CSS styles in your project that may set that color?

Can you share a reproduction link?

**Reply by ErfanRez** [reply]
No I 'm not using any CSS styles at all it's all daisy UI. I just put a useEffect on a swap button to change the data-theme of root element to either dark or light. here is a live production link: https://dejavu-cms.vercel.app

**Reply by saadeghi** [reply, marked as answer]
`data-theme="dark"` is added to the div  with `root` id, not the root `<html>` element so because that div doesn't have enough content to fill the screen, you see this issue.  
You can either set `data-theme` to the root `<html>` element, or you can add a `class="min-h-screen"` to the div  with `root` id so it fills the height.

**Reply by ErfanRez** [reply]
Yes of course. Thankyou for your guidance.🙏
